utilities/oadp.py

```
@@ -137,3 +207,138 @@ def create_rhel_vm(
+        complete_timeout (int):
+            Timeout (seconds) to wait for restore COMPLETED when wait_complete is True.
+        **kwargs (Any):
+            Additional keyword arguments passed to the base ``Restore`` class.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        client: DynamicClient,
+        complete_timeout: int = TIMEOUT_5MIN,
+        wait_complete: bool = True,
```

Ruff (0.14.11)

Boolean-typed positional argument in function definition

(FBT001)